src/data/models/Happiness.ts

```diff
@@ -1,7 +1,8 @@
-import { User } from "./User";
-
 export interface Happiness {
-  score: number;
+  id: number;
+  user_id: number;
```

Comment:
Eventually this will need to be refactored to a User object for author, but it's fine for now I guess, unless you want to pre-emptively refactor it and wait for Aaron's PR.